react-native/Fable.Helpers.ReactNative.fs

```diff
@@ -137,12 +463,16 @@ module Props =
         | BorderTopWidth of float
         | BorderWidth of float
         | Bottom of float
-        | Flex of int
-        | FlexDirection of string
-        | FlexWrap of string
+        | Flex of float
```

Comment:
Not sure, but looking at https://github.com/facebook/css-layout/tree/master/csharp/Facebook.CSSLayout the values are floats, which is consistent with other values, like margin.